pois/generate_pois.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import os
+import subprocess
+
+
+# This tool takes an England-wide osm.pbf (from
+# http://download.geofabrik.de/europe/great-britain/england-latest.osm.pbf) and
+# generates files with point-of-interest layers for ATIP.
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("input", help="Path to england-latest.osm.pbf file", type=str)
+    args = parser.parse_args()
+
+    # https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dschool indicates
+    # primary and secondary schools
+    generatePolygonAmenity(args, "school", "schools")
+
+    # Note https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dhospital doesn't
+    # cover all types of medical facility
+    generatePolygonAmenity(args, "hospital", "hospitals")
+
+
+# Extract `amenity={amenity}` polygons from OSM, and only keep a name attribute.
+def generatePolygonAmenity(args, amenity, filename):
+    # Remove files from any previous run
+    try:
+        os.remove(f"{filename}.osm.pbf")
+        os.remove(f"{filename}.geojson")
+        os.remove(f"{filename}.pmtiles")
+    except:
+        pass
+
+    # First extract a .osm.pbf with all amenity={name} features
+    # TODO Do we need nwr? We don't want points further on
+    run(
+        [
+            "osmium",
```

Comment:
There's a variety of toolchains and libraries in different languages for extracting OSM data. For simple cases of grabbing geometric objects matching some tags, osmium is dirt simple and extremely performant